modules/images-configuration-blocked-payload.adoc

```diff
@@ -0,0 +1,48 @@
+//Modules included in the following assemblies
+//
+// * openshift_images/image-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="images-configuration-blocked-payload"]
+
+= Allow blocking of a payload registry
+
+When your mirrors are already configured using a ImageContentSourcePolicy (ICSP) object, you can block upstream payload registries in a disconnected environment. 
+
+.Procedure
+
+. Mirror the payload to a registry in your use case using an ImageContentSourcePolicy (ICSP) object. The following example ICSP mirrors the payload `itnernal-mirror.io/openshift-payload`.
++
+[source, yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: my-icsp
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - internal-mirror.io/openshift-payload
+    source: quay.io/openshift-payload
+
+----
+. Once the object deploys onto your nodes, use the following command to edit the `image.config.openshift.io`.
++
+[source, terminal]
+----
+oc edit `image.config.openshift.io` cluster
+----
+. To block the payload registry, add the following configuration to the file. 
++
+[source, yaml]
+----
+spec:
+  registrySource:
+    blockedRegistries:
+     - quay.io/openshift-payload
+
+----
+. Check that the new configuration is applied to the nodes.
+
+
```

Comment:
Is there a command/example output we can add here so that users can verify that the configuration is applied?

Reply:
They can check the output of the `/etc/containers/registries.conf` file on the node to see that the upstream payload registry is now blocked.